### PR TITLE
Add Bulgarian language translations.

### PR DESCRIPTION
**Pull request by ghost**
Hello. I added the Bulgarian language translations to NeoInfo.. 2nd commit was to fix a small inconsistency in a few of the words in the About page with my translations..

### CODE REVIEW

File: app/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: You don't need the first line

File: app/src/main/res/values-bg/strings.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="general">Общо</string>
+    <string name="camera">Камера</string>
+    <string name="misc">Разни</string>
+    <string name="bootloader">Буутлоудър</string>
+    <string name="baseband">Базов Диапазон/Модем</string>
+    <string name="device_variant">Вариант на устройтството</string>
+    <string name="android_ver">Версия на Android</string>
+    <string name="android_api">API версия на Android</string>
+    <string name="build_number">Номер на компилация</string>
+    <string name="rear_samsung_sensor">"Сензор на Samsung "</string>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: Please remove the quotes and the whitespace at the end

File: app/src/main/res/values-bg/strings.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="rear_sony_sensor">Сензор на Sony</string>
+    <string name="rear_unspecific_sensor">Неспецифично</string>
+    <string name="fw_rear_camera">"Фърмуер "</string>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: Same as above

File: app/src/main/res/values-bg/strings.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="kernel_rear_camera">Ядро</string>
+    <string name="vendor_rear_camera">Вендор</string>
+    <string name="kernel_front_camera">"Ядро (отпред) "</string>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: Same as above

File: app/src/main/res/values-bg/strings.xml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="rear_camera_status">Състояние на задната камера</string>
+    <string name="cam_status_okay">Работещо</string>
+    <string name="cam_status_warn">Вендорът е инициализиран, но сензорът на камерата може да не работи</string>
+    <string name="cam_status_fail">"Не работи "</string>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: Same as above

File: app/src/main/res/values-bg/strings.xml [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="unknown">Непознато</string>
+    <string name="general">Общо</string>
+    <string name="camera">Камера</string>
+    <string name="misc">Разни</string>
+    <string name="bootloader">Буутлоудър</string>
+    <string name="baseband">Базов Диапазон/Модем</string>
```
Comment (prototype74) [inline app/src/main/res/values-bg/strings.xml]: I don't know bulgarian but what do you think about this?

```suggestion
    <string name="baseband">Основната честота/Модем</string>
```

**Review by prototype74 — CHANGES_REQUESTED** [review]
Thanks for you contribution!
I can't actually check if it's correctly translated but I think it looks okay. Therea are still some minor things that bother me. Please check and fix them so I can merge your PR.

**Review by AnoUser99 — APPROVED** [review]

**Comment by ghost** [comment]
I updated it but seems like we cannot update the 0 line.

File: app/src/main/res/values-bg/strings.xml (hunk 1 above)
Comment (ghost) [reply app/src/main/res/values-bg/strings.xml]: Resolved.

**Review by prototype74 — APPROVED** [review]

**Pull request merged**